tasks/configure.js

```diff
@@ -44,9 +44,36 @@ function fetchFromVault (source, target, serviceData) {
 			}, {});
 
 			return Object.assign({}, shared, appVars.data);
+		})
+		.then((keys) => {
+			const { TEST_USER_TYPES, TEST_SESSIONS_URL, TEST_SESSIONS_API_KEY } = keys;
+			if (!(TEST_USER_TYPES && TEST_SESSIONS_API_KEY && TEST_SESSIONS_URL)) {
+				return keys;
+			}
+
+			const userTypes = TEST_USER_TYPES.split(',');
+			return Promise.all(userTypes.map((type) => appendSessionToken(keys, type.trim().toLowerCase(), TEST_SESSIONS_URL, TEST_SESSIONS_API_KEY)))
+				.then(() => keys)
+				.catch((e) => {
+					console.log('Couldn\'t fetch the session tokens. Please check TEST_USER_TYPES, TEST_SESSIONS_URL and TEST_SESSIONS_API_KEY environment variables.'); // eslint-disable-line no-console
+					console.error(e); // eslint-disable-line no-console
+				});
 		});
 }
 
+const appendSessionToken = (keys, userType, url, apiKey) => {
```

Comment:
might be nicer if this function didn't mutate keys, but returned something that gets added to keys while it's in scope above?